ads/google/a4a/docs/a4a-readme.md

```diff
@@ -28,21 +28,26 @@ AMP ads are designed to work on both AMP and Non-AMP webpages,  including deskto
 
 # Current status
 
-The AMP ads format spec has been [released](https://github.com/google/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md) and any creative developer can create AMP ads. In order for ads to get preferred treatment on AMP pages, ad server support is required. Advertisers using DoubleClick or  AdSense can  already deliverprogrammatic AMP ads to publisher AMP pages. Publishers using  DFP (DoubleClick for Publishers) can already deliver their own AMP ads. Advertisers or publishers using other ad providers  can implement AMP ads with the help of a signing service like CloudFlare. Cloudflare provides AMP ad verification services, enabling any independent ad provider to deliver faster, lighter, and more engaging ads.
+The AMP ads format spec has been [released](https://github.com/google/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md) and any creative developer can create AMP ads.A number of ad providers are working on automatically converting ads to AMP ads whenever possible. e.g. Adsense
```

Comment:
Add spaces between "create AMP ads."  and "A number".  Also I believe the S is capitalized in AdSense